### ISSUE TITLE
cursive dropcap gets clipped

### ISSUE THREAD

**Issue by 5e-Cleric** [naturalcrit/homebrewery#3114]
As [reported on reddit](https://www.reddit.com/r/homebrewery/comments/17gaerj/comment/k6nbsbt/?context=3&sort=new), the clip box of the drop cap letter does not change based on text variants or styles.

This means that setting the text as cursive makes the letter clip behind text:

<img width="359" alt="image" src="https://github.com/naturalcrit/homebrewery/assets/91662974/c9d77f4a-cb90-4cc7-bc12-a8bb55f979b7">

**Comment by G-Ambatte** [comment]
I have so far been unable to develop a CSS selector to target the first letter only when it is styled. However I sturdier the easiest solution is to instead simply remove any such styling from the drop cap, so it is never italicized.

**Comment by 5e-Cleric** [comment]
you can't style based on applied styles, unless they are set in inline styles:

`.page .element[style*="color:red;"] {}` should work fine.

But that does not work for the `::first-letter` pseudo-selector.

A solution would be to fix this for manual cursive, but not for programatic one:

```
# This is a test

*T*his test will try to find a solution for the drop cap clip when you use cursive.
```

This one outputs an `em` element inside the P, and can be styled easily. And somehow counts the T as first-letter, so it can be styled with `.page h1+p:has(em)::first-letter`.

**Comment by G-Ambatte** [comment]
I was attempting to use something like `.page h1+p:has(em:first-child)::first-letter` to target only first letters that were italicized, but it seems that the text elements inside a `p` don't count as children, so ANY `p` that contained an `em` was targeted.

**Comment by Gazook89** [comment]
Is it easier to create a distinct font variant of that drop cap font, italic? Then could bake in better spacing. Rather than relying on browser to fake the italic.

**Comment by G-Ambatte** [comment]
I suspect that the easiest solution is simply to remove the italicization, with the primary justification that no drop cap in any official source book is italicized, and secondary justifications that the kerning between two different fonts is difficult to manage, even more so with additional text styling.

**Comment by Gazook89** [comment]
K. I also agree that italicizing the drop cap makes no sense. I don’t think we need to do anything here— just tell users not to include it in the italic syntax.

**Comment by 5e-Cleric** [comment]
I could give a try to making an italicized version, but i agree there is no point in having that option.

**Comment by Gazook89** [comment]
I don’t think we should enforce anything. It’s easy enough to just not include the letter in the italics. If we have weird hidden rules it’s only going to confuse people who want to do something a little different, with different fonts or whatever.

**Comment by dbolack-ab** [comment]
Did this get resolved?

**Comment by 5e-Cleric** [comment]
This did get stuck as is seen in the issue, the issue was small enough to not even get to gitter chat.

I now think this should be "fixed" by adding a "set cursive drop cap" snippet that includes the extra space for it.

**Comment by 5e-Cleric** [comment]
I have changed my mind, this is a non-issue, should be resolved as themes for specific books or even specific pages, not as a rule. (even if we could)
